pyproject.toml

```diff
@@ -1,9 +1,13 @@
 [tool.ruff]
 select = [
   "ASYNC",  # flake8-async
+  "C4",     # flake8-comprehensions
+  "C90",    # McCabe code complexity
```

Comment:
I actually prefer not to use this; IMO it's a net-negative for projects that have a good culture of human code review 🙂